editors/neovim-qt/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+PortGroup github 1.0
+
+# use cmake 1.0 for CMAKE_BUILD_TYPE=Release
+PortGroup cmake 1.0
```

Comment:
Please don't; use the latest version of the PortGroup and set it to "Release" in the appropriate way - check the PG in how to do that are check other ports.